tests/core/job/test_job_repository.py

```diff
@@ -64,15 +64,16 @@ def h():
         pass
 
 
-job = Job(JobId("id"), task)
+job = Job(JobId("id"), task, "submit_id")
 job._subscribers = [f, A.f, A.g, A.h, A.B.f]
-job._exceptions = [traceback.TracebackException.from_exception(Exception())]
+# job._exceptions = [traceback.TracebackException.from_exception(Exception())]
```

Comment:
Why this change?

Reply:
It wasn't used anywhere or for any purposes. I'm thinking of removing it

Reply:
If we don't store it, the developer will not be able to use it for instance to debug or to show it to the end-user in a UI.

Reply:
ah I see, sure!